edx_reverification_block/xblock/reverification_block.py

```diff
@@ -29,25 +29,25 @@ class ReverificationBlock(XBlock):
     # Fields
     display_name = String(
         scope=Scope.settings,
-        default='Re-Verification Checkpoint',
+        default='Reverification Checkpoint',
         help="This name appears in the horizontal navigation at the top of "
              "the page."
     )
 
     attempts = Integer(
-        display_name="Verification Attempts",
+        display_name="Verification attempts",
         default=0,
         scope=Scope.settings,
-        help="This is the number of attempts that students are permitted to "
-             "get a valid re-verification."
+        help="The number of times learners can attempt to verify their identity. "
+             "This number can be 1, 2, or 3."
```

Comment:
Delete "This number can be 1, 2, or 3." - when I was doing these text revisions, there was a blank field instead of a drop-down list, so we needed the specificity.